tracee-rules/main.go

```diff
@@ -51,28 +63,24 @@ func main() {
 		Flags: []cli.Flag{
 			&cli.StringSliceFlag{
 				Name:  "rules",
-				Usage: "select which rules to load",
+				Usage: "select which rules to load as a comma seperated list, use --list for rules to select from",
 			},
 			&cli.StringFlag{
 				Name:  "rules-dir",
-				Usage: "directory where to search for rules",
-			},
-			&cli.StringFlag{
-				Name:  "webhook",
-				Usage: "call this HTTP endpoint for every match",
-			},
-			&cli.StringFlag{
-				Name:  "tracee-file",
-				Usage: "path to Tracee Gob output file",
-			},
-			&cli.StringFlag{
-				Name:  "stdin-as",
-				Usage: "read events from stdin and treat them as JSON serialized events of the specified input source. this will override an already configured input source",
+				Usage: "directory where to search for rules in OPA (.rego) or Go plugin (.so) formats",
 			},
 			&cli.BoolFlag{
 				Name:  "list",
 				Usage: "print all available rules",
 			},
+			&cli.StringFlag{
+				Name:  "webhook",
+				Usage: "HTTP endpoint to call for every match",
+			},
+			&cli.StringSliceFlag{
+				Name:  "input-tracee",
+				Usage: "specify various key:value pairs for input options from tracee-ebpf, use '--input-tracee help' for more info",
```

Comment:
may I suggest an alternate text:
`configure tracee-ebpf as input source. see '--input-tracee help' for more info`